--- a/Rendering/Matplotlib/vtkMatplotlibMathTextUtilities.h
+++ b/Rendering/Matplotlib/vtkMatplotlibMathTextUtilities.h
@@ -143,6 +143,10 @@
   // is called internally when matplotlib rendering is first needed and is used
   // to implement IsAvailable.
   static Availability CheckMPLAvailability();
+  static void DisableMPL()
+    {
+    MPLMathTextAvailable = UNAVAILABLE;
+    };
 
   // Description:
   // Cache the availability of matplotlib in the current python session.
