Index: VTK-6.2.0/Rendering/Matplotlib/vtkMatplotlibMathTextUtilities.h
===================================================================
--- VTK-6.2.0.orig/Rendering/Matplotlib/vtkMatplotlibMathTextUtilities.h
+++ VTK-6.2.0/Rendering/Matplotlib/vtkMatplotlibMathTextUtilities.h
@@ -132,6 +132,10 @@ protected:
   // Function used to check MPL availability and update MPLMathTextAvailable.
   // This will do tests only the first time this method is called.
   static void CheckMPLAvailability();
+  static void DisableMPL()
+    {
+    MPLMathTextAvailable = UNAVAILABLE;
+    };
 
 private:
   vtkMatplotlibMathTextUtilities(const vtkMatplotlibMathTextUtilities&); // Not implemented.
