--- a/Utilities/Doxygen/doxyfile.in
+++ b/Utilities/Doxygen/doxyfile.in
@@ -12,7 +12,7 @@
 GENERATE_TODOLIST    = YES
 GENERATE_BUGLIST     = YES
 GENERATE_HTML        = YES
-GENERATE_HTMLHELP    = YES
+GENERATE_HTMLHELP    = NO
 GENERATE_LATEX       = NO
 GENERATE_MAN         = NO
 GENERATE_RTF         = NO
@@ -92,14 +92,16 @@
 SORT_MEMBER_DOCS     = NO
 DISTRIBUTE_GROUP_DOC = YES
 TAB_SIZE             = 3
-DETAILS_AT_TOP       = YES
+#obsolete
+#DETAILS_AT_TOP       = YES
 SHORT_NAMES          = @DOXYGEN_SHORT_NAMES@
 # PAPER_TYPE           = letter
 
 FILE_PATTERNS        = *.h
 RECURSIVE            = YES
 
-HTML_ALIGN_MEMBERS   = YES
+#obsolete
+"HTML_ALIGN_MEMBERS   = YES
 
 ALPHABETICAL_INDEX   = YES
 COLS_IN_ALPHA_INDEX  = 3
