Index: vtk-6.0.0/Examples/GUI/Python/OrthogonalPlanesWithTkPhoto.py
===================================================================
--- vtk-6.0.0.orig/Examples/GUI/Python/OrthogonalPlanesWithTkPhoto.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/GUI/Python/OrthogonalPlanesWithTkPhoto.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import vtk
 from vtk import *
 import Tkinter
Index: vtk-6.0.0/Examples/Graphics/Python/SegmentAndBrokenLineSources.py
===================================================================
--- vtk-6.0.0.orig/Examples/Graphics/Python/SegmentAndBrokenLineSources.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Graphics/Python/SegmentAndBrokenLineSources.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 ############################################################
 from vtk import *
 ############################################################
Index: vtk-6.0.0/Examples/Graphics/Python/SingleYoungsMaterialInterface.py
===================================================================
--- vtk-6.0.0.orig/Examples/Graphics/Python/SingleYoungsMaterialInterface.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Graphics/Python/SingleYoungsMaterialInterface.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 ############################################################
 from vtk import *
 from vtk.util.misc import vtkGetDataRoot
Index: vtk-6.0.0/Examples/Hybrid/Python/CylinderAndPolarAxes.py
===================================================================
--- vtk-6.0.0.orig/Examples/Hybrid/Python/CylinderAndPolarAxes.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Hybrid/Python/CylinderAndPolarAxes.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 ############################################################
 from vtk import *
 ############################################################
Index: vtk-6.0.0/Examples/Infovis/Python/Matlab_engine_filter.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/Matlab_engine_filter.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/Matlab_engine_filter.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 
 # Python example script that uses the Matlab Engine filter to compute the
 # degree centraility for an input graph and display the result.
Index: vtk-6.0.0/Examples/Infovis/Python/Matlab_engine_interface.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/Matlab_engine_interface.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/Matlab_engine_interface.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 
 # Python example script that uses the vtkMatlabEngineInterface to perform
 # a calculation (sin(x)^2 + cos(x)^2 = 1) on VTK array data, and pass the
Index: vtk-6.0.0/Examples/Infovis/Python/Rcalculator_mst.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/Rcalculator_mst.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/Rcalculator_mst.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 
 # Python examples script that uses the R calculator filter to find the
 # maxiumum spanning tree of a random input graph by inverting the edge
Index: vtk-6.0.0/Examples/Infovis/Python/Rcalculator_vd.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/Rcalculator_vd.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/Rcalculator_vd.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 
 # Python example script that uses the RcalculatorFilter to compute the mean
 # vertex degree of the input graph and the distance from the mean for each
Index: vtk-6.0.0/Examples/Infovis/Python/Rinterface.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/Rinterface.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/Rinterface.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 
 # Python example script that uses the vtkRIinterface to create an instance of the
 # R interpreter and pass it some data, modify the data in R, and pass the result
Index: vtk-6.0.0/Examples/Infovis/Python/Rrand_table.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/Rrand_table.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/Rrand_table.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 
 # Python example script that uses the Gnu R random table source to create an output
 # table with 5 columns and 30 rows.  The columns are filled random numbers drawn
Index: vtk-6.0.0/Examples/Infovis/Python/adjacency_matrix_to_graph.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/adjacency_matrix_to_graph.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/adjacency_matrix_to_graph.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 import os.path
 
Index: vtk-6.0.0/Examples/Infovis/Python/aggregate.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/aggregate.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/aggregate.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 import os.path
 
Index: vtk-6.0.0/Examples/Infovis/Python/boost_bfs.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/boost_bfs.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/boost_bfs.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 source = vtkRandomGraphSource()
Index: vtk-6.0.0/Examples/Infovis/Python/boost_centrality.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/boost_centrality.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/boost_centrality.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 source = vtkRandomGraphSource()
Index: vtk-6.0.0/Examples/Infovis/Python/boost_mst.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/boost_mst.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/boost_mst.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 source = vtkRandomGraphSource()
Index: vtk-6.0.0/Examples/Infovis/Python/boost_mst_extract_graph.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/boost_mst_extract_graph.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/boost_mst_extract_graph.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 source = vtkRandomGraphSource()
Index: vtk-6.0.0/Examples/Infovis/Python/boost_mst_with_hgv.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/boost_mst_with_hgv.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/boost_mst_with_hgv.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 source = vtkRandomGraphSource()
Index: vtk-6.0.0/Examples/Infovis/Python/circular_vertices.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/circular_vertices.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/circular_vertices.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 """
 This file demonstrates drawing graphs using circular vertices via
 vtkRenderedGraphRepresentation.
Index: vtk-6.0.0/Examples/Infovis/Python/cone_layout.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/cone_layout.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/cone_layout.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 
Index: vtk-6.0.0/Examples/Infovis/Python/contingency_port_protocol.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/contingency_port_protocol.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/contingency_port_protocol.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 import os.path
 
Index: vtk-6.0.0/Examples/Infovis/Python/create_tree.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/create_tree.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/create_tree.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 graph = vtkMutableDirectedGraph()
Index: vtk-6.0.0/Examples/Infovis/Python/database_query1.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/database_query1.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/database_query1.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 import os.path
 
Index: vtk-6.0.0/Examples/Infovis/Python/databases.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/databases.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/databases.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 import os.path
 
Index: vtk-6.0.0/Examples/Infovis/Python/databases2.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/databases2.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/databases2.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 import os.path
 
Index: vtk-6.0.0/Examples/Infovis/Python/delimited_text_reader1.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/delimited_text_reader1.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/delimited_text_reader1.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 csv_source = vtkDelimitedTextReader()
Index: vtk-6.0.0/Examples/Infovis/Python/delimited_text_reader2.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/delimited_text_reader2.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/delimited_text_reader2.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 csv_source = vtkDelimitedTextReader()
Index: vtk-6.0.0/Examples/Infovis/Python/displayVTKHierarchy.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/displayVTKHierarchy.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/displayVTKHierarchy.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import vtk
 import inspect
 import csv
Index: vtk-6.0.0/Examples/Infovis/Python/filters.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/filters.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/filters.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 source = vtkRandomGraphSource()
Index: vtk-6.0.0/Examples/Infovis/Python/geoview.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/geoview.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/geoview.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 import os.path
 
Index: vtk-6.0.0/Examples/Infovis/Python/graph1.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/graph1.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/graph1.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 """
 This file demonstrates the creation of a directed graph using the
 Python interface to VTK.
Index: vtk-6.0.0/Examples/Infovis/Python/graph3d.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/graph3d.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/graph3d.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 from vtk.util.misc import vtkGetDataRoot
 VTK_DATA_ROOT = vtkGetDataRoot()
Index: vtk-6.0.0/Examples/Infovis/Python/graph_selection_subtract.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/graph_selection_subtract.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/graph_selection_subtract.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 source = vtkRandomGraphSource()
Index: vtk-6.0.0/Examples/Infovis/Python/graph_tree_ring.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/graph_tree_ring.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/graph_tree_ring.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 addStringLabel = vtkProgrammableFilter()
Index: vtk-6.0.0/Examples/Infovis/Python/gui.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/gui.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/gui.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from PyQt4 import QtCore
 from PyQt4 import QtGui
 from PyQt4 import uic
Index: vtk-6.0.0/Examples/Infovis/Python/hello_world.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/hello_world.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/hello_world.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 
 from vtk import *
 
Index: vtk-6.0.0/Examples/Infovis/Python/hierarchical_graph.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/hierarchical_graph.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/hierarchical_graph.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 source = vtkRandomGraphSource()
Index: vtk-6.0.0/Examples/Infovis/Python/icicle_view.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/icicle_view.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/icicle_view.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 reader2 = vtkXMLTreeReader()
Index: vtk-6.0.0/Examples/Infovis/Python/icicle_view_simple.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/icicle_view_simple.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/icicle_view_simple.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 reader1 = vtkXMLTreeReader()
Index: vtk-6.0.0/Examples/Infovis/Python/labels.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/labels.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/labels.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 
 import random
 from vtk import *
Index: vtk-6.0.0/Examples/Infovis/Python/merge_graphs.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/merge_graphs.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/merge_graphs.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 """
 This file demonstrates the creation of a directed graph using the
 Python interface to VTK.
Index: vtk-6.0.0/Examples/Infovis/Python/mysql_database.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/mysql_database.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/mysql_database.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 database = vtkSQLDatabase.CreateFromURL("mysql://enron@vizdb.srn.sandia.gov:3306/enron")
Index: vtk-6.0.0/Examples/Infovis/Python/programmable_pipeline.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/programmable_pipeline.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/programmable_pipeline.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 source = vtkRandomGraphSource()
Index: vtk-6.0.0/Examples/Infovis/Python/programmable_pipeline2.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/programmable_pipeline2.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/programmable_pipeline2.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 """
 This file demonstrates the creation of a directed graph using the
 Python interface to VTK.
Index: vtk-6.0.0/Examples/Infovis/Python/psql_database.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/psql_database.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/psql_database.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 # Network Database
Index: vtk-6.0.0/Examples/Infovis/Python/random3d.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/random3d.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/random3d.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 source = vtkRandomGraphSource()
Index: vtk-6.0.0/Examples/Infovis/Python/selection.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/selection.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/selection.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 source = vtkRandomGraphSource()
Index: vtk-6.0.0/Examples/Infovis/Python/selection_domain.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/selection_domain.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/selection_domain.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 import os.path
 
Index: vtk-6.0.0/Examples/Infovis/Python/simple_selection.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/simple_selection.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/simple_selection.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 from vtk import *
Index: vtk-6.0.0/Examples/Infovis/Python/statistics_temperatures.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/statistics_temperatures.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/statistics_temperatures.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 import os.path
 
Index: vtk-6.0.0/Examples/Infovis/Python/streaming_statistics.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/streaming_statistics.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/streaming_statistics.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 import os.path
 
Index: vtk-6.0.0/Examples/Infovis/Python/streaming_statistics_pyqt.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/streaming_statistics_pyqt.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/streaming_statistics_pyqt.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 import os.path
 import sys
Index: vtk-6.0.0/Examples/Infovis/Python/tables1.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/tables1.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/tables1.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 """
 This file contains Python code illustrating the creation and manipulation of
 vtkTable objects.
Index: vtk-6.0.0/Examples/Infovis/Python/tables2.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/tables2.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/tables2.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 """
 This file contains Python code illustrating the creation and manipulation of
 vtkTable objects.
Index: vtk-6.0.0/Examples/Infovis/Python/tables3.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/tables3.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/tables3.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 if __name__ == "__main__":
Index: vtk-6.0.0/Examples/Infovis/Python/tables4.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/tables4.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/tables4.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 """
 This file provides a more advanced example of vtkTable access and
 manipulation methods.
Index: vtk-6.0.0/Examples/Infovis/Python/tables_adv.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/tables_adv.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/tables_adv.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 """
 This file contains Python code illustrating the creation and manipulation of
 vtkTable objects.
Index: vtk-6.0.0/Examples/Infovis/Python/theme.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/theme.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/theme.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 source = vtkRandomGraphSource()
Index: vtk-6.0.0/Examples/Infovis/Python/tree1.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/tree1.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/tree1.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 """
 This file demonstrates the creation of a tree using the
 Python interface to VTK.
Index: vtk-6.0.0/Examples/Infovis/Python/tree_map.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/tree_map.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/tree_map.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 import os.path
 
Index: vtk-6.0.0/Examples/Infovis/Python/treemap_view.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/treemap_view.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/treemap_view.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 reader1 = vtkXMLTreeReader()
Index: vtk-6.0.0/Examples/Infovis/Python/treering_hierarchical_view.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/treering_hierarchical_view.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/treering_hierarchical_view.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 
 from vtk import *
 
Index: vtk-6.0.0/Examples/Infovis/Python/treering_view.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/treering_view.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/treering_view.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 reader2 = vtkXMLTreeReader()
Index: vtk-6.0.0/Examples/Infovis/Python/treering_view_simple.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/treering_view_simple.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/treering_view_simple.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 reader1 = vtkXMLTreeReader()
Index: vtk-6.0.0/Examples/Infovis/Python/vertex_degree_programmable.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/vertex_degree_programmable.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/vertex_degree_programmable.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 
 vertexDegree = vtkProgrammableFilter()
Index: vtk-6.0.0/Examples/Infovis/Python/views.py
===================================================================
--- vtk-6.0.0.orig/Examples/Infovis/Python/views.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/Infovis/Python/views.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 import os.path
 
Index: vtk-6.0.0/Examples/ParallelProcessing/MPI/Python/ParallelCone.py
===================================================================
--- vtk-6.0.0.orig/Examples/ParallelProcessing/MPI/Python/ParallelCone.py	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/Examples/ParallelProcessing/MPI/Python/ParallelCone.py	2013-10-18 23:11:25.000000000 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 from vtk import *
 import sys
 import os
