Index: vtk-6.0.0/Rendering/Matplotlib/vtkMatplotlibMathTextUtilities.h
===================================================================
--- vtk-6.0.0.orig/Rendering/Matplotlib/vtkMatplotlibMathTextUtilities.h	2013-10-28 14:28:13.000000000 +0100
+++ vtk-6.0.0/Rendering/Matplotlib/vtkMatplotlibMathTextUtilities.h	2013-10-28 14:30:17.000000000 +0100
@@ -117,6 +117,10 @@
   // Function used to check MPL availability and update MPLMathTextAvailable.
   // This will do tests only the first time this method is called.
   static void CheckMPLAvailability();
+  static void DisableMPL()
+    {
+    MPLMathTextAvailable = UNAVAILABLE;
+    };
 
 private:
   vtkMatplotlibMathTextUtilities(const vtkMatplotlibMathTextUtilities&); // Not implemented.
