Index: vtk-6.0.0/CMake/vtkTclWrapping.cmake
===================================================================
--- vtk-6.0.0.orig/CMake/vtkTclWrapping.cmake	2013-06-12 21:47:10.000000000 +0200
+++ vtk-6.0.0/CMake/vtkTclWrapping.cmake	2013-10-24 11:30:33.000000000 +0200
@@ -46,6 +46,11 @@
   foreach(dep ${${vtk-module}_LINK_DEPENDS})
     if(NOT "${vtk-module}" STREQUAL "${dep}")
       if(NOT ${dep}_EXCLUDE_FROM_WRAPPING)
+	if("${${dep}_TCL_NAME}" STREQUAL "")
+	  #message(STATUS "Tcl libdepends for " ${vtk-module} " APPEND extra_links " ${dep} "->" ${${dep}_TCL_NAME}TCL)
+	  #set(${dep}_TCL_NAME "${dep}")
+	  vtk_module(${dep})
+	endif()
         list(APPEND extra_links ${${dep}_TCL_NAME}TCL)
       endif()
     endif()
